tests/data/importJsonResume.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { BasicsStore, type Education, type Work } from '@src/data/data';
+import { beforeAll, describe, expect, it } from 'vitest';
+import { type Writable, writable, get } from 'svelte/store';
+import importJsonResume from '@src/data/importJsonResume';
+import type { JsonResume } from '@src/data/JsonResume';
+
+const testResumeJson: JsonResume = {
```

Comment:
Could I just  have this in a separate file and then import it into this one? That would be nice for using it in both unit tests and when doing manual testing.